solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -16,184 +15,101 @@ import {IVestingEscrowSimple} from 'interfaces/IVestingEscrowSimple.sol';
-   *        offsetted to start from `Sept 5th 2024 - 1 month`: At Sept 5th 2024 the cliff
-   *        is triggered unlocking 1/13 of the tokens, and then 1/13 of the tokens will
-   *        be linearly unlocked every month after that.
-   */
-
-  /**
-   * @notice Vesting duration including one month of cliff
-   * @dev 13 months duration
-   */
+  /// @inheritdoc IConnextVestingWallet
   uint64 public constant VESTING_DURATION = ONE_YEAR + ONE_MONTH;
 
-  /**
-   * @notice Vesting cliff duration
-   * @dev 1 month cliff
-   */
+  /// @inheritdoc IConnextVestingWallet
   uint64 public constant VESTING_CLIFF_DURATION = ONE_MONTH;
 
-  /**
-   * @notice Vesting warmup time
-   * @dev 11 months offset
-   */
+  /// @inheritdoc IConnextVestingWallet
   uint64 public constant VESTING_OFFSET = ONE_YEAR - ONE_MONTH;
 
-  /**
-   * @notice Vesting start date
-   * @dev Sept 5th 2024 - 1 month
-   */
+  /// @inheritdoc IConnextVestingWallet
   uint64 public constant VESTING_START_DATE = NEXT_TOKEN_LAUNCH + VESTING_OFFSET;
 
-  /*///////////////////////////////////////////////////////////////
-                             STORAGE
-  //////////////////////////////////////////////////////////////*/
-
-  /**
-   * @notice Total amount of tokens to be vested
-   * @dev Set into constructor
-   */
+  /// @inheritdoc IConnextVestingWallet
   uint256 public immutable TOTAL_AMOUNT;
 
-  /**
-   * @dev Init VestingWalletWithCliff and save total amout of tokens to be unlocked
-   * @param _beneficiary  The address of the beneficiary
-   * @param _totalAmount  The total amount of tokens to be unlocked
-   */
-  constructor(
-    address _beneficiary,
-    uint256 _totalAmount
-  ) VestingWalletWithCliff(_beneficiary, VESTING_START_DATE, VESTING_DURATION, VESTING_CLIFF_DURATION) {
-    TOTAL_AMOUNT = _totalAmount;
-  }
-
-  /*///////////////////////////////////////////////////////////////
-                            ERRORS
-  //////////////////////////////////////////////////////////////*/
-
-  /**
-   * @notice Permission denied
-   */
-  error NotAllowed();
-
-  /**
-   * @notice Zero address not allowed
-   */
-  error ZeroAddress();
+  /// @inheritdoc IConnextVestingWallet
+  uint64 public immutable CLIFF;
 
-  /*///////////////////////////////////////////////////////////////
-                            OVERRIDES
-  //////////////////////////////////////////////////////////////*/
+  /// @inheritdoc IConnextVestingWallet
+  uint256 public released;
 
   /**
-   * @inheritdoc VestingWallet
-   * @notice This contract is only meant to unlock NEXT tokens
+   * @param _owner The address of the beneficiary
+   * @param _totalAmount The total amount of tokens to be unlocked
    */
-  function vestedAmount(uint64) public view virtual override returns (uint256 _amount) {
-    return 0;
+  constructor(address _owner, uint256 _totalAmount) Ownable(_owner) {
+    CLIFF = VESTING_START_DATE + VESTING_CLIFF_DURATION;
```

Comment:
CLIFF is basically `VESTING_START_DATE + 1 year` no?

Reply:
> CLIFF is basically `VESTING_START_DATE + 1 year` no?

 `VESTING_START_DATE + 1 month`. It's used just once so we could replace it with `VESTING_START_DATE + VESTING_CLIFF_DURATION` in the code

Reply:
please do 🙏

Reply:
> VESTING_CLIFF

I'd remove it at all or moved its declaration to constants. Setting constants in constructor doesn't make sence